experimental/ratify/proto/v1/orchestrator.proto

```diff
@@ -0,0 +1,58 @@
+syntax="proto3";
+
+package orchestrator;
+
+option go_package = "github.com/deislabs/ratify/experimental/proto/v1/orchestrator";
+
+import "common.proto";
+import "google/protobuf/struct.proto";
+
+service PluginOrchestrator {
+    rpc GetBlobsForArtifact (GetBlobsForArtifactRequest) returns (GetBlobsForArtifactResponse);
+    rpc GetManifest (GetManifestRequest) returns (GetManifestResponse);
+}
+
+message GetBlobsForArtifactRequest {
+    common.ArtifactDescriptor artifact = 1;
+}
+
+message GetBlobsForArtifactResponse {
+    common.ArtifactDescriptor artifact = 1;
+    bytes content = 2;
```

Comment:
will this return the bytes content for all the blobs in a given artifact? Is the returned `content` serialized to be an array where each member contains bytes for each blob?